yeonjae/baekjoon/입출력연습/2439-별찍기2.js

```diff
@@ -0,0 +1,25 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const num = Number(input.toString().split(" "));
+  let answer = "";
+  let blank = "";
+
+  for (let i = 1; i <= num; i++) {
+    answer += "*";
+    for (let j = 0; j < num - i; j++) {
+      blank += " ";
+    }
+    console.log(blank + answer);
+    blank = "";
+  }
```

Comment:
```suggestion
  const num = Number(input.toString().split(" "));
  let answer = "";
  let blank = "";

  for (let i = 1; i <= num; i++) {
    answer += "*";
    for (let j = 0; j < num - i; j++) {
      blank += " ";
    }
    console.log(blank + answer);
    blank = "";
  }
```
- 콘솔로그를 반복문마다 5번찍는게 좋을까요??ㅎㅎㅎ
- 아래와 같이 padStart, repeat를 이용한 풀이도 가능합니다
```js
  const n = +input[0];
  let result = "";

  for (let i = 1; i <= n; i++) {
    const repeatedStar = "*".repeat(i);
    result += repeatedStar.padStart(n, " ") + "\n";
  }

  console.log(result);
```

Reply:
padStart도 다음에 한번꼭 사용해보겠습니다!!!ㅎㅎㅎ